nym-wallet/src-tauri/src/coin.rs

```diff
@@ -171,34 +157,45 @@ impl Coin {
     self.amount.clone()
   }
 
+  #[allow(unused)]
   pub fn denom(&self) -> Denom {
     self.denom.clone()
   }
-}
 
-impl TryFrom<Coin> for CosmWasmCoin {
-  type Error = BackendError;
+  // Helper function that returns the local denom in terms of the specified network denom.
+  fn denom_as_string(&self, network_denom: &CosmosDenom) -> Result<String, BackendError> {
```

Comment:
I wonder if having separate Minor/Major denoms even make sense as in cosmos there's only one name used for all coins (say `unym` and `nym` is only ever used for UI/UX purposes)

Reply:
Yes I think we should consider removing that capability, only store in Minor denom and just have the option to display in Major when needed